services/121-service/test/program/create-program.test.ts

```diff
@@ -84,35 +87,7 @@ describe('Create program', () => {
     // Assert
     // const programId = createProgramResponse.body.id;
     expect(createProgramResponse.statusCode).toBe(HttpStatus.BAD_REQUEST);
-
-    // A new program should not have been created
-    expect(getProgramResponse.statusCode).toBe(HttpStatus.NOT_FOUND);
-  });
-
-  it.skip('should not be able to post a program with double names', async () => {
```

Comment:
why is this test removed?

Reply:
It's not right? These is still a `  it('should not be able to post a program with 2 of the same names', async () => {` it's just at line ~54